zerver/webhooks/circleci/view.py

```diff
@@ -118,10 +118,10 @@ def get_commit_details(payload: WildValue) -> str:
             )
 
         branch = payload["pipeline"]["vcs"]["branch"].tame(check_none_or(check_string))
-        commit_subject = payload["pipeline"]["vcs"]["commit"]["subject"].tame(
+        commit_topic = payload["pipeline"]["vcs"]["commit"]["subject"].tame(
```

Comment:
This change is not required. `commit_subject` makes more sense as that is what's being extracted from the payload.